jdaviz/configs/default/plugins/gaussian_smooth/tests/test_gaussian_smooth.py

```diff
@@ -20,10 +20,10 @@ def test_linking_after_spectral_smooth(spectrum1d_cube):
 
     assert len(dc) == 2
     assert dc[1].label == 'Smoothed test stddev 3.2'
-    assert len(dc.external_links) == 1
+    assert len(dc.external_links) == 3
```

Comment:
Why is this changed? This PR does not touch any Gaussian Smooth stuff, does it?

Reply:
This PR adds more links (pixel x and y) for cubes in addition to linking for moment map, so that is where the extra 2 `external_links` come from.